frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -8,13 +8,13 @@ type SizeMap = Array<SizeMapEntry>;
 
 // These should be sorted in order of ascending width.
 const guideImageSizeMap: SizeMap = [
-   { name: 'mini', width: 56 },
-   { name: 'thumbnail', width: 96 },
-   { name: '140x105', width: 140 },
-   { name: '200x150', width: 200 },
-   { name: 'standard', width: 300 },
-   { name: '440x330', width: 440 },
-   { name: 'medium', width: 592 },
+   { name: 'mini', width: 41 },
+   { name: 'thumbnail', width: 70 },
+   { name: '140x105', width: 110 },
```

Comment:
I think I'm missing something here. Why did we change the sizes of these images? I feel like this is supposed to be our knowledge of what size these images are. Changing them to not reflect reality seems like a bad idea.